infrastructure/cluster/flux-v2/jhub/jhub-release.yaml

```diff
@@ -125,6 +125,10 @@ spec:
                 echo "account = $JUPYTERHUB_USER" >> /opt/rucio/etc/rucio.cfg;
                 echo "auth_type = oidc" >> /opt/rucio/etc/rucio.cfg;
                 echo "oidc_audience = rucio" >> /opt/rucio/etc/rucio.cfg;
+                echo "oidc_scope = openid profile offline_access" >> /opt/rucio/etc/rucio.cfg;
+                echo "oidc_issuer = escape" >> /opt/rucio/etc/rucio.cfg;
+                echo "auth_oidc_refresh_active true" >> /opt/rucio/etc/rucio.cfg;
+                echo "auth_oidc_refresh_before_exp 20" >> /opt/rucio/etc/rucio.cfg;
```

Comment:
The syntax of lines 130 and 131 are not exactly the same (there is no `=` sign). Is this correct or just an oversight ?

Reply:
According to the [rucio](https://rucio.cern.ch/documentation/user/using_the_client) docs this is the correct syntax.